step.yml

```diff
@@ -223,6 +223,13 @@ inputs:
         The token required to authenticate with the API.
       is_dont_change_value: true
       is_sensitive: true
+  - generate_universal_apk_if_none: "false"
+    opts:
+      title: "Force generate universal APK for AABs"
```

Comment:
Hi @lchrislee ! Thank you for contribution. Please make the default value of this input true so it will not be a breaking change on the step's contract.

Reply:
Good catch! Thanks for looking into this.

On a side note, I have no idea how unit tests passed when I defaulted to false 🤷. I'd love to update the unit tests but I haven't had time to look into my setup since opening the PR.

Reply:
Hi! Unfortunately not all of our steps are covered fully with tests, we're continuously looking into improving our steps and trying to prioritize work based on their usage and importance. We're planning on tackling the real issue defined in #93 so this flag is just a temporary measure. You can just go with setting the flag true. We'll open an other issue to clean the apk generation.